src/GovUk.Education.ExploreEducationStatistics.Data.Model/Observation.cs

```diff
@@ -22,7 +22,7 @@ public class Observation
 
         public Dictionary<Guid, string> Measures { get; set; }
 
-        public ICollection<ObservationFilterItem> FilterItems { get; set; }
+        public List<ObservationFilterItem> FilterItems { get; set; }
```

Comment:
I'm all for switching to Lists, but just interested to know what the driver was here and in a few other places?

Reply:
Think this might actually be a straggler from my next PR (which depends on this one)!

I've reverted it back to `ICollection`